### ISSUE TITLE
Keyboard navigation is missing

### ISSUE THREAD

**Issue by Steve-A-Orr** [quist00/onesearch-local#11]
This was reported by one of our library staff recently:

> The OneSearch searchbox on the library homepage is not navigable using the keyboard. 
> 
> If you start typing "climate change", and want to use the the keyboard arrows (up/down) to choose Articles, Books, or CSU+, the searchbox does not respond. From an accessibility standpoint, this makes the searchbox difficult to use.
> 
> The searchbox hosted on the Primo homepage does not exhibit this problem and behaves correctly. I'm asking that this can be brought to the attention of the original developer or troubleshooted locally.

**Comment by Steve-A-Orr** [comment]
It seems that you can navigate to the selections/scopes by **tabbing**. It takes two tab clicks (first, submit icon; then, advanced search button) to get to the list, from the point of inserting text.

For some reason, when the insert is in the form text field, the arrows don't accomplish any action; but when the focus has gone on to the form buttons or scope list, the arrows control the browser view...and so it moves the page up and down.

So, even though the arrows work in the new AngularJS Primo UI, I don't know if it can be called a _bug_ for it not to work in this Angular2 app.

**Comment by quist00** [comment]
this one is important. I will try to address it quickly

**Comment by quist00** [comment]
addressed in f544f729e37eae19b100554d76c1d57887dc8d55
